### ISSUE TITLE
Make it less confusing to use Cupertino with text under MaterialApp

### ISSUE THREAD

**Issue by xster** [flutter/flutter#12845]
Avoid user confusing wrt the default error text style from MaterialApp when using Cupertino widgets.

**Comment by Hixie** [comment]
Can you elaborate on this issue?

Based on the description so far it doesn't sound like it fits our current priority list so I'm moving this to the next milestone, but I'm not 100% sure I understand the issue.

**Comment by Hixie** [comment]
Right. If you're using a MaterialApp, you're expected to use material widgets and conventions, and one of those conventions is text only paints on Material widgets. This is quite intentional.

**Comment by Piinks** [comment]
This issue is being closed as the framework'\''s behavior regarding text styling and inheritance is intentional and well-documented. `MaterialApp` is designed to provide Material-specific defaults, and mixing in `Cupertino` widgets requires explicit theme management or the use of `CupertinoTheme`. The current design ensures that each design system remains consistent with its own conventions.

If there is more work to do here, please let us know by filing a new issue with up to date information. Thanks!
